doc/websocket.qbk

```diff
@@ -152,23 +159,23 @@ been upgraded. The HTTP Upgrade request must include the Host HTTP field,
 and the URI of the resource to request. `hanshake` is used to send the
 request with the required host and resource strings.
 ```
-    websocket::stream<ip::tcp::socket> ws(ios);
+    beast::websocket::stream<boost::asio::ip::tcp::socket> ws(ios);
     ...
-    ws.set_option(websocket::keep_alive(true));
+    ws.set_option(beast::websocket::keep_alive(true));
     ws.handshake("ws.mywebapp.com:80", "/cgi-bin/bitcoin-prices");
 ```
 
-The `websocket::stream` automatically handles receiving and processing
-the HTTP response to the handshake request. The call to handshake is
-successful if a HTTP response is received with the 101 "Switching Protocols"
-status code. On failure, an error is returned or an exception is thrown.
-Depending on the keep alive setting, the socket may remain open for a
-subsequent handshake attempt
+The [link beast.ref.websocket__stream `beast::websocket::stream`] automatically
+handles receiving and processing the HTTP response to the handshake request.
+The call to handshake is successful if a HTTP response is received with the
```

Comment:
This is a super-minor nit, and I get that reasonable people can view this topic differently - just pointing it out: "an HTTP response" reads better than "a HTTP response".